test/xpu_api/random/interface_tests/engines_methods.pass.cpp

```diff
@@ -202,7 +202,6 @@ test_vec<oneapi::dpl::ranlux24_vec<N>>
                         engine1.discard(offset);
                         typename oneapi::dpl::ranlux24_vec<N>::result_type res0;
                         oneapi::dpl::ranlux24_vec<N> engine(engine1);
-                        auto eng = engine.base();
```

Comment:
This line shouldn't be removed as it is a simple check of API. Like that .base() is callable from this engine.

Reply:
Fixed.

Reply:
@paveldyakov , then I believe you should check somehow this `eng` variable.

Is not is better to leave it as is (without `[[maybe_unused]]`) to indicate there is something you should fix?

Reply:
@rarutyun, the API call is also fine. If API doesn't exist we will have the error on compilation stage.